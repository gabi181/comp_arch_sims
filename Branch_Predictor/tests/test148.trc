32 1 6 3 local_history global_tables using_share_mid
0x3FA4AD44 N 0x081B933C
0x4AA11E84 T 0x23258DD0
0x35D51644 T 0x375B2228
0x2A6600F8 N 0x35C02088
0x6E1E5E9C N 0x413AD184
0x35D51644 N 0x375B2228
0x6E1E5E9C T 0x413AD184
0x5046C47C T 0x5F8F22A4
0x069C1EBC T 0x52BB8584
0x3FA4AD44 N 0x081B933C
0x2A6600F8 N 0x35C02088
0x41F75B60 T 0x37C18520
0x4AA11E84 T 0x23258DD0
0x069C1EBC T 0x52BB8584
