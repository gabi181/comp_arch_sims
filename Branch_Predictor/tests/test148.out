0x3fa4ad44 N 0x3fa4ad48
0x4aa11e84 N 0x4aa11e88
0x35d51644 N 0x35d51648
0x2a6600f8 N 0x2a6600fc
0x6e1e5e9c N 0x6e1e5ea0
0x35d51644 N 0x35d51648
0x6e1e5e9c N 0x6e1e5ea0
0x5046c47c N 0x5046c480
0x69c1ebc N 0x69c1ec0
0x3fa4ad44 N 0x3fa4ad48
0x2a6600f8 T 0x35c02088
0x41f75b60 N 0x41f75b64
0x4aa11e84 N 0x4aa11e88
0x69c1ebc T 0x52bb8584
flush_num: 8, br_num: 14, size: 1220b
